server/index.js

```diff
@@ -1,13 +1,61 @@
 import express from "express";
 import mongoose from "mongoose";
 import cors from "cors";
+import UserModel from "./models/User";
```

Comment:
Here also its giving me an error you just need to add the .js so:
`import UserModel from "./models/User.js";`